doc/admin-guide/plugins/statichit.en.rst

```diff
@@ -0,0 +1,106 @@
+                                               --disable-exact, and URL path is appended to this directory path to get
+                                               the path for the file containing the body.  If the specified file is not a
+                                               directory, it should contain the body.
+--mime-type        -m       No         Yes     Defaults to "text/plain".
+--max-age          -a       No         Yes     Defaults to "0".  If the value is not 0, it gives the number of seconds the
+                                               static body will remain fresh when cache.  (Including in the proxy's cache, if
+                                               enabled.)
+--failure-code     -c       No         Yes     Defaults to "404" (NOT FOUND).  This response status will be returned if the
+                                               file which should contain the body does not exist.  This option is
+                                               particularly useful when --file-path specifies a directory.
+--success-code     -s       No         Yes     Defaults to "200" (OK). This is the response status when the static object is
+                                               successfully returned in the transaction response.
+--disable-exact    -d       No         No      Disable "exact" URL match.  "Exact" match, for this plugin, means the URL must
+                                               not include a path.
+=================  =======  =========  ======  ==============
+
+When using long option names, the value can be specified by appending = and then the value.  `@pparam=--max-age=5` for
+example.  Otherwise, the value must be specified as the next pparam.  `@pparam=-a @pparam=5` for example.
+
+When exact match is enabled, if the request URL has a path, a response with NOT FOUND status will be sent.
+
+Mime Headers in Responses with OK/200 Status
+============================================
+
+#.  Content-Type.  The value given with the --mime-type parameter, or its default value.
+#.  Content-Length.
+#.  Cache-Control.  If --max-age is 0, the value will be "no-cache".  Otherwise, the value will be "max-age=d", where
+    d stands for the value given with the --max-age parameter.
+#.  if --max-age is not 0, Last-Modified is present, and gives the time the static body was last cached.
+
+Metrics
+=======
+
+The plugin publishes the following metrics:
+
+======================== ====================================================
+Name                     Description
+======================== ====================================================
+statichit.response_bytes The total number of bytes emitted
+statichit.response_count The number of HTTP responses generated by the plugin
+======================== ====================================================
+
+remap.config Examples:
+======================
+
```

Comment:
Serve a request for /internal/healthcheck with the contents of the file located at /var/run/trafficserver/healthcheck.txt. If the file is present, send with a Mime-type of text/plain. Als set the Cache-Control: header to no-cache.